inventory_reporting/reports/inventory_report.py

```diff
@@ -127,6 +127,11 @@ def generate_storable_report(self, workbook, wizard):
                     ("stock_move_id.picking_code", "in", ("internal", "outgoing")),
                     ("stock_move_id.unbuild_id", "=", False),
                     ("stock_move_id.origin_returned_move_id", "=", False),
+                    (
+                        "stock_move_id.location_id.is_subcontracting_location",
+                        "=",
+                        False,
+                    ),
```

Comment:
Shouldn't it be like this:

```suggestion
                    "|",
                    "&",
                    ("stock_move_id.location_dest_id.usage", "=", "production"),
                    (
                        "stock_move_id.location_id.is_subcontracting_location",
                        "=",
                        False,
                    ),
                    "&",
                    ("stock_move_id.location_id.usage", "=", "production"),
                    (
                        "stock_move_id.location_dest_id.is_subcontracting_location",
                        "=",
                        False,
                    ),
                    ("stock_move_id.picking_code", "in", ("internal", "outgoing")),
                    ("stock_move_id.unbuild_id", "=", False),
                    ("stock_move_id.origin_returned_move_id", "=", False),
```